src/containers/Results/index.js

```diff
@@ -132,54 +86,95 @@ class Results extends Component {
           <SearchInput onSubmit={this.handleSubmit} searchTerm={searchTerm} />
         </div>
 
-        {isLoading || this.state.isLoading ? (
-          <div className="loader" />
-        ) : !results.length ? (
-          <EmptyStates searchTerm={searchTerm} />
-        ) : (
-          <div className="results__container">
-            <div className="results__top-bar">
-              {results.length ? (
-                <div className="results__number-results">
-                  <NumberOfResults />
-                </div>
-              ) : null}
-
-              <DataSetSampleActions
-                data={samplesAsDataSet}
-                enableAddRemaining={false}
-                meta={{
-                  buttonStyle: 'secondary',
-                  addText: 'Add Page to Dataset'
-                }}
+        <Loader
```

Comment:
Not really for this PR: I found the rendering logic here kind of confusing. Maybe we should add some comments to explain it?